Devcon4_Security_Agenda.md

```diff
@@ -2,19 +2,72 @@
 
 ## October 30th
 
+- EF Grants Overview
+	- Kenneth Ng, 12:10PM - 12:20PM, Side Stage
+		
 - Less Gas, More Fun: Optimising Smart Contracts through Yul
     - Christian Reitwiessner, 1:40PM - 1:50PM, Side Stage
 
+- The Trinity Ethereum Client - A platform for blockchain applications
+	- Piper Merriam, 3:30PM - 3:40PM, Side Stage
+
+- Build your own Multi-Sig Wallet in Two Hours
+	- Dan Nolan, Cody Mccabe, 1:30 PM - 3:30PM, Coding Workshop
+
 
 ## October 31sth
 
+- Satoshi Has No Clothes: Failures in On-Chain Privacy
+	- Ian Miers, 2:00PM - 2:25PM, Side Stage
+
+- Browser 3.0 - How to Build Secure Web3 Clients
+	- Philipp Langhans, 2:30PM - 2:55PM, Side Stage
+
+- SpankChain: Payment Channels in Production
+	Ameen Soleimani, Arjun Bhupati, 3PM - 3:25PM, Side Stage
+
+- Less Gas, More Fun: Optimising Smart Contracts through Yul
+	- Christian Reitwiessner, 4:30PM - 4:55PM, Side Stage
+
+- P4: Private Periodic Payments Protocol
+	- Liz Steininger, 4:00PM - 4:25PM, Breakout Room Presentations
 
 ## Novemer 1st
 
+- Security Breakout: Live Smart Contract Hacking
+	- Martin Holst Swende, Nick Johnson, Richard Moore, Matthew Di Ferrante, 12:30PM - 1:30PM, Breakout Room
+
+- Reversing Ethereum Smart Contracts to find out what's behind EVM bytecode
+	- Patrick Ventuzelo, 3:00PM - 3:25PM, Breakout Room
 
+- P2P Networking in Ethereum 2.0
+	- Jannik Luhn, Kevin Mai-Hsuan Chia, 1:30PM - 1:55PM, Side Stage
+
+- MakerDAO: A New Hope for Financial Reform
+	- Andy Milenius, 2PM - 2:30PM, Auditorium
+
+- Human Centered Identity: We are more than keys
+	- Jonny Howle, 4:35PM - 5:00PM, Side Stage
```

Comment:
At 4:55 pm on this day is a session called "Current State of Security Panel", so it appears that this wasn't done very thoroughly.

Reply:
i tried to keep around 4 events per day and i didn't get the topic properly so, sorry for that !!